.github/codecov.yml

```diff
@@ -5,6 +5,15 @@ coverage:
   precision: 2
   round: down
   range: "80...100"
+  status:
+    patch:
+      default:
+        informational: true
+    project:
+      default:
+        base: auto
+        target: auto
+        threshold: 2%
```

Comment:
@mdonadoni Changes to code coverage settings were necessary due to [codecov updates](https://about.codecov.io/blog/codecov-is-updating-its-github-integration/). The values above are one possibility how to make CI pass and yet encourage some code coverage. Let's discuss IRL tomorrow what targets and thresholds we would like to aim at.